torch/onnx/symbolic_opset9.py

```diff
@@ -2466,7 +2732,7 @@ def _unique2(g, input, sorted, return_inverse, return_counts):
 
 # TODO(justinchuby): Clean up this function generation magic by defining the functions
 # explicitly.
-for k, v in symbolic_helper.cast_pytorch_to_onnx.items():
+for k, v in symbolic_helper.cast_pytorch_to_onnx.items():  # type: ignore[has-type]
```

Comment:
curious why this change is needed?

Reply:
I am not sure exactly. mypy complains with this change. My guess is because we added more typing contexts to the code, mypy decided to start reporting errors.